packages/sanity/src/desk/i18n/resources.ts

```diff
@@ -165,6 +165,136 @@ const deskLocaleStrings = {
 
   /** --- "PRODUCTION PREVIEW", eg link to content --- */
   'production-preview.menu-item.title': 'Open preview',
+
+  /** -- DESK PANES -- */
+  /** The tool tipe for the split pane button on the document panel header */
+  'document-panel-header.split-pane-button.tooltip': 'Split pane right',
```

Comment:
We want to avoid "locations" in keys such as "header" and "footer", as these elements may move around.